source/guides/clear/clear-linux-dissector.rst

```diff
@@ -0,0 +1,302 @@
+   later imports.
+
+#. To import the latest |CL| release, check the :guilabel:`Get latest` box.
+
+#. To import a specific release, enter the release number in the
+   :guilabel:`Release` field.
+
+#. Click :guilabel:`IMPORT` and you will be taken to a page that displays the task
+   status. It will take a long time to upload the |CL| data.
+
+#. Once the |CL| data is uploaded, you will see a page similar to the following:
+
+   .. figure:: ../../_figures/clear-linux-dissector/distro-data-import-final.png
+      :scale: 45%
+      :alt: Finished distribution data import
+
+      Figure 2: Finished distribution data import
+
+Import Yocto Project Data
+=========================
+
+Yocto Project data is imported from https://layers.openembedded.org.
+
+Import Yocto Project Data via CLI
+---------------------------------
+
+To import the latest Yocto Project data from the master branch:
+
+* Run the following command:
+
+  .. code-block:: bash
+
+      docker-compose run --rm layersapp /opt/layerindex/layerindex/tools/import_layers.py https://layers.openembedded.org
+
+To import Yocto Project data from a specific branch:
+
+#. First create a branch object in the admin page.
+
+   #. Navigate to https://localhost:8081/admin/layerindex/branch/add/ and enter the
+      branch name in the :guilabel:`Branch name` field.
+
+   #. Enter the corresponding Bitbake branch name (for example "zeus") in the
+      :guilabel:`Bitbake branch` field.
+
+      .. figure:: ../../_figures/clear-linux-dissector/create-yoctoproject-branch.png
+         :scale: 45%
+         :alt: Importing Yocto Project data from branch
+
+         Figure 3: Importing Yocto Project data from branch
+
+
+#. Run the following command to import a specific Yocto Project branch:
+
+   .. code-block:: bash
+
+       docker-compose run --rm layersapp /opt/layerindex/layerindex/tools/import_layers.py https://layers.opnembedded.org -b zeus
+
+To update your Yocto Project branches:
+
+* Run the following command:
+
+  .. code-block:: bash
+
+     docker-compose run --rm layersapp /opt/layerindex/layerindex/update.py
+
+View Distribution Data
+**********************
+View the distribution data of a specific |CL| and Yocto Project distribution
+side-by-side with the `Distro data` tool.
+
+#. From the homepage, click on :guilabel:`Distro data`.
+
+   .. figure:: ../../_figures/clear-linux-dissector/distro-data-select.png
+      :scale: 45%
+      :alt: Distro data button
+
+      Figure 4: Distro data button
+
+#. In the :guilabel:`Branch` section in the top left of the screen, select which
+   imported |CL| distribution you want to search in, then search for the package
+   you'd like to include, as shown in Figure 5:
+
+   .. figure:: ../../_figures/clear-linux-dissector/distro-data-form.png
+      :scale: 45%
+      :alt: Select distribution to search
+
+      Figure 5: Select distribution to search
+
+#. Select the package you want to include.
```

Comment:
include -> examine perhaps? I'm not sure we're including anything at this stage.